.flowconfig

```diff
@@ -31,6 +31,7 @@ module.name_mapper='^rewards\(.*\)$' -> '<PROJECT_ROOT>/ui/rewards\1'
 module.name_mapper='^i18n\(.*\)$' -> '<PROJECT_ROOT>/ui/i18n\1'
 module.name_mapper='^effects\(.*\)$' -> '<PROJECT_ROOT>/ui/effects\1'
 module.name_mapper='^comments\(.*\)$' -> '<PROJECT_ROOT>/ui/comments\1'
```

Comment:
Firefox Android doesn't seem to have the combo box to select?
iOS Safari seems ok.
<img width="687" alt="image" src="https://user-images.githubusercontent.com/64950861/155736332-671fd1f8-c7cd-45ac-be22-3707c4dafd11.png">

Reply:
Chrome Android ok too
<img width="400" alt="image" src="https://user-images.githubusercontent.com/64950861/155736919-d0b0a21f-e7a5-4203-be6b-8a22fcebeca8.png">

Reply:
wait but that pic only has the language option?

Reply:
Ah, oops.  The site must be randomizing the locations.